recipes-test/demo-network-config/network-config.inc

```diff
@@ -0,0 +1,19 @@
+CONF_TYPE ?= "${@ 'multihomed' if d.getVar('MACHINE') == 'raspberrypi3' and d.getVar('IMAGE_TYPE', '') == 'primary' else 'static'}"
```

Comment:
Why the empty string for the second parameter of `d.getVar('IMAGE_TYPE', '')`?

Reply:
The empty string is used as a default value of 'IMAGE_TYPE' in case if it's not defined.  If the default value is not specified (d.getVar('IMAGE_TYPE')) then an error will be yielded if 'IMAGE_TYPE' is not defined.

Reply:
I've just replaced d.getVar('IMAGE_TYPE', '') with d.getVar('IMAGE_TYPE') as it doesn't bring any value here.